client/packages/coldchain/src/Sensor/Components/SensorLineForm.tsx

```diff
@@ -71,6 +71,12 @@ export const SensorLineForm: FC<UseDraftSensorControl> = ({
           draft.latestTemperatureLog?.nodes[0]?.datetime
         )}
       />
+      <TextWithLabelRow
+        sx={textSx}
+        labelProps={labelWrap}
+        label={t('label.sensor-type')}
+        text={draft.type ?? ''}
```

Comment:
maybe this should be in a utils file, as a helper function? though to get `useTranslation` it would have to be in a hook. you could pass `t` to the helper function but that's messy IMHO